### DISCUSSION TITLE
DeoVR Alternative?

### DISCUSSION THREAD

**Discussion by MrChristopher360** [xbapps/xbvr#715] [General]
DeoVR has stopped working for me for reasons I will leave up to their devs and support. Browsing their forum though I know that I am far from alone.

That leaves this amazing project basically unusable unless there is another way to browse the content in VR. Does an alternate method exist?

**Comment by theRealKLH** [comment]
for non-script usage, check out HereSphere... There is finally a Quest native app over on Sidequest.
